src/FTPWindow.cpp

```diff
@@ -851,17 +849,14 @@ void TFTPWindow::CopyUrl()
 		= ((BStringField *)row->GetField(CLM_INTERNAL_NAME))->String();
 
 	BString url;
-	if (fCurrentRemoteDir.Length() > 0 &&
-		fCurrentRemoteDir[fCurrentRemoteDir.Length() - 1] == '/') {
-
-		url << "ftp://" << fHost << ":" << fPort
-			<< fCurrentRemoteDir << intName;
-	} else {
-		url << "ftp://" << fHost << ":" << fPort
-			<< fCurrentRemoteDir << "/" << intName;
+	url << "ftp://" << fHost << ":" << fPort << fCurrentRemoteDir;
+	if (fCurrentRemoteDir.Length() <= 0
+		|| fCurrentRemoteDir[fCurrentRemoteDir.Length() - 1] != '/') {
+		url << "/";
 	}
+	url << intName;
 
-	BMessage *clip = (BMessage *)NULL;
+	BMessage *clip = NULL;
```

Comment:
```suggestion
	BMessage* clip = NULL;
```

Reply:
Yeah, the problem is that the codebase does not follow the guidelines most of the time.
And so every pointer in the code is like 'Type *variablename' and not 'Type* variablename'
even though it should be.

Reply:
So If I modified it, the coding style would not be uniform.

Reply:
It's already not consistent, even just in this file. I think the rule of thumb is that you fix the coding style (at least) for the lines you touch.